### PR TITLE
Add ChartDB

### PR DESCRIPTION
**Pull request by johnnyfish**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

File: software/chartdb.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  - Docker
+tags:
+  - Database Management
+  - Analytics
+  - Software Development - IDE & Tools
+source_code_url: https://github.com/chartdb/chartdb
+demo_url: https://app.chartdb.io/examples
+stargazers_count: 3833
+updated_at: "2024-10-27"
+archived: false
```
Comment (kokomo123) [inline software/chartdb.yml:16]:
```suggestion
```
This will be maintained automatically, no need for these lines 👍🏻

Comment (johnnyfish) [reply software/chartdb.yml]: Oops. tnx, removing it now.

Comment (johnnyfish) [reply software/chartdb.yml]: @kokomo123 fixed that. any other issues? 
Thank you for your help!

Comment (kokomo123) [reply software/chartdb.yml]: One last thing. Unfortunately, it seems there are no tagged releases. Rather then pull from main/latest all the time, users should be able to verify that its stable and have changelogs, etc. We can check back in around 4 months from the first stable tagged release.

Comment (johnnyfish) [reply software/chartdb.yml]: @kokomo123 Thank you for pointing that out! I didn’t realize the importance of having tagged releases for stability and changelogs. I’ll make adjustments to remove the related updates to tags and focus solely on adding ChartDB in this PR, which should help streamline the review process. Would it be possible to push this quickly?

Comment (kokomo123) [reply software/chartdb.yml]: Unfortunately, it is not possible to push through PRs quickly (as far as im aware of according to the requirements/guidelines). Usually if you start making tagged releases now and we wait 4 months, we can then reopen the PR, and promptly approve it and merge it within a week.

Comment (johnnyfish) [reply software/chartdb.yml]: @kokomo123 what? let's close this PR then, and I will open a new PR without a tagged releases, why to wait 4 months now?

Comment (kokomo123) [reply software/chartdb.yml]: I'll have @nodiscc @Ki-er explain it better as they may have a better explaination then what i'm saying.

Comment (johnnyfish) [reply software/chartdb.yml]: @kokomo123 Any update on that? :)

Comment (nodiscc) [reply software/chartdb.yml]: I just added the "standard" reply to this frequent issue here: https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/CONTRIBUTING.md#no-tagged-releases

**Comment by nodiscc** [comment]
Hi, thanks for your contribution.

However, there are no tagged releases for this project. Our guidelines require that Any software project you are adding was first released more than 4 months ago. We encourage you to create a release now and/or a simple [changelog](https://keepachangelog.com/en/1.1.0/) that will help users keep track of changes in the software (especially breaking changes or changes requiring configuration tweaks), and will allow administrators to install a known working, unchanging version (as opposed to always installing the latest development version).

Once this is done, the project may be resubmitted to awesome-selfhosted when the first release reaches the age of 4 months.

Thanks for understanding, and good luck with this project.

**Pull request closed without merging**